calculator.html

```diff
@@ -3,75 +3,80 @@
 
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Calculator</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <link rel="stylesheet" type="text/css" href="calculator.css">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/knockout/3.4.0/knockout-min.js"></script>
 </head>
 
 <body>
-  <div class="container" data-bind="foreach: calculators">
-    <div class="main-body"> <span> calculator </span>
-      <form>
-        <div class="input-div">
-          <input type="text" id="result" data-bind="value: input" />
-        </div>
-        <ul class="first-row">
-          <li>
-            <button data-bind="click: addNumber.bind(null, '7')">7</button>
-          </li>
-          <li>
-            <button data-bind="click: addNumber.bind(null, '8')">8</button>
-          </li>
-          <li>
-            <button data-bind="click: addNumber.bind(null, '9')">9</button>
-          </li>
-          <li>
-            <button id="clear" data-bind="click: clear">C</button>
-          </li>
-        </ul>
-        <ul class="second-row">
-          <li>
-            <button data-bind="click: addNumber.bind(null, '4')">4</button>
-          </li>
-          <li>
-            <button data-bind="click: addNumber.bind(null, '5')">5</button>
-          </li>
-          <li>
-            <button data-bind="click: addNumber.bind(null, '6')">6</button>
-          </li>
-          <li>
-            <button id="plus" data-bind="click: addPlus">+</button>
-          </li>
-        </ul>
-        <ul class="third-row">
-          <li>
-            <button data-bind="click: addNumber.bind(null, '1')">1</button>
-          </li>
-          <li>
-            <button data-bind="click: addNumber.bind(null, '2')">2</button>
-          </li>
-          <li>
-            <button data-bind="click: addNumber.bind(null, '3')">3</button>
-          </li>
-          <li>
-            <button id="subtraction" data-bind="click:addMinus">-</button>
-          </li>
-        </ul>
-        <ul class="fourth-row">
-          <li>
-            <button id="zero" data-bind="click: addNumber.bind(null, '0')">0</button>
-          </li>
-          <li>
-            <button id="multiply" data-bind="click: addMultiplication">&times;</button>
-          </li>
-          <li>
-            <button id="div" data-bind="click: addDivision">&divide;</button>
-          </li>
-          <li>
-            <button id="equal" data-bind="click: equal">=</button>
-          </li>
-        </ul>
-      </form>
-      <a href="#" data-bind="click: $parent.removeCalculator, visible: $parent.calculators().length > 1">Remove</a>
+  <div class="container stay-in-centre col-lg-2 col-mid-5 col-sm-12 col-xs-12">
+    <div data-bind="foreach: calculators" class="foreach" >

```

Comment:
You can use a virtual binding for the calculators. Then you won't have to nest everything inside an additional div (which is what has caused this big diff). It would look something like `<!-- ko foreach: calculators -->` and the closing tag `<!-- /ko -->` instead of `<div data-bind="foreach: calculators">` and `</div>`